### PR TITLE
Bug 1993985, Bug 1993980, Bug 1990041: duplicated events: update list of known issues

### PR DESCRIPTION
**Pull request by stbenjam**
Several have been fixed, and we've identified a few new ones.

### CODE REVIEW

File: pkg/synthetictests/duplicated_events.go [hunk 1]
```diff
@@ -100,14 +96,23 @@ var knownEventsBugs = []knownProblem{
...
+	{
+		Regexp: regexp.MustCompile(`ns/.*horizontalpodautoscaler.*failed to get cpu utilization: unable to get metrics for resource cpu: no metrics returned from resource metrics API.*`),
+		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1993985",
+	},
+	{
+		Regexp: regexp.MustCompile(`ns/.*unable to ensure pod container exists: failed to create container.*slice already exists.*`),
+		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1993980",
+	},
+	{
+		Regexp: regexp.MustCompile(`ns/openshift-etcd-operator deployment/etcd-operator - reason/UnhealthyEtcdMember unhealthy member.*`),
```
Comment (deads2k) [inline pkg/synthetictests/duplicated_events.go:112]: is this happening outside of upgrade jobs?

Comment (stbenjam) [reply pkg/synthetictests/duplicated_events.go]: No and I see you already handled this in https://github.com/openshift/origin/commit/de8df4de72a40bc6bc561dc9562d49d50ec45e54

I've removed it

**Pull request merged**